src/web-auth/index.js

```diff
@@ -339,24 +348,37 @@ WebAuth.prototype.validateAuthenticationResponse = function(
         }
       );
     }
+
     if (
       validationError.error !== 'invalid_token' ||
-      validationError.errorDescription === 'Nonce does not match.'
+      (validationError.errorDescription &&
```

Comment:
Consider breaking this logic out into a variable for more clarity. 

Also ... why are we checking the description as well if we hav an error code for this?

Reply:
This just modifies existing logic rather than rewriting it. We do have an error code but it's the same error code for every error. I don't think the check here is terrible, but it could be refactored later if desired.